mrtt-ui/src/data/questions.js

```diff
@@ -172,6 +172,20 @@ const restorationAims = {
   }
 }
 
+const causesOfDecline = {
```

Comment:
continuing from other comment; This could be 
```
const causesofDecline = {
   causesOfDeclineOptions: causesOfDeclineOptions (imported)
   ...
   }
   ```